idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/annotators/CoercionAnnotatorTest.kt

```diff
@@ -40,7 +45,11 @@ class CoercionAnnotatorTest : IdeTestSetUp(
             collectAnnotations(code, myFixture)
           },
           result = resolvesWhen("CoercionAnnotatorTest2 for 1 implicit coercion property") { result: List<HighlightInfo> ->
-            result.size == 1
+            result.size == 1 &&
+              result[0].forcedTextAttributes == implicitProofAnnotatorTextAttributes &&
+              result[0].severity == HighlightSeverity.INFORMATION &&
+              result[0].quickFixActionRanges[0].first.action.text.contains("Go to proof: consumer.handle") &&
```

Comment:
I suppose this is not supposed to be hardcoded

Reply:
nevermind, read it out of context and didn't realize it was a test

Reply:
Yeah, I was checking the content of that annotation for that specific use case 👍